herbie_google_pubsub/apps.py

```diff
@@ -0,0 +1,11 @@
+from django.apps import AppConfig
+from herbie_core.services.message_publisher.registry import Registry
+from herbie_google_pubsub.publisher import GooglePubsubPublisher
```

Comment:
is the application ready here for import or maybe you get exception?

Reply:
I think it should be ready, I didn't get an exception yet. But good point I will doublecheck.